### PR TITLE
Update MIME type for glb files

### PR DESCRIPTION
**Pull request by emackey**
This was changed in KhronosGroup/glTF#943.

Also fixed an eslint warning in `server.js`.

### CODE REVIEW

**Comment by mramato** [comment]
Model.js has a list of accepted model mime types that also needs to be updated.

**Comment by emackey** [comment]
Thanks @mramato.

**Comment by mramato** [comment]
For the Model.js change, are we okay with only requesting the official mime types and not the older versions?  Technically that's a breaking change (though I doubt many people rely on it).

**Comment by emackey** [comment]
We've got `*/*` on the end there, so the old types aren't completely removed, they're just down at the bottom of the priority list.

File: Source/Scene/Model.js [hunk 1]
```diff
@@ -154,8 +154,8 @@ define([
         FAILED : 3
     };
 
-    // GLTF_SPEC: Figure out correct mime types (https://github.com/KhronosGroup/glTF/issues/412)
-    var defaultModelAccept = 'model/vnd.gltf.binary,model/vnd.gltf+json,model/gltf.binary,model/gltf+json;q=0.8,application/json;q=0.2,*/*;q=0.01';
+    // GLTF_SPEC: Figure out correct mime types (https://github.com/KhronosGroup/glTF/issues/412 and https://github.com/KhronosGroup/glTF/issues/943)
```
Comment (mramato) [inline Source/Scene/Model.js]: Sorry, last comment.  Is this comment still needed?  1.0 and 2.0 specs are done and all mime types are known, right?

**Comment by emackey** [comment]
Good point about that comment.  I wanted to keep the links, but I changed the rest of the wording to sound more final and less like a TODO.

**Comment by emackey** [comment]
This is ready.

**Comment by mramato** [comment]
Whoops, thought I merged this.

**Comment by mramato** [comment]
Thanks @emackey!

**Pull request merged**